README.md

```diff
@@ -1,141 +1,23 @@
-<h1 align="center">Web-dev-mini-projects</h1> 
-
-<div align="center">
-<a href="https://github.com/topics/html"><img alt="HTML" src="https://img.shields.io/badge/HTML%20-%23E34F26.svg?&style=for-the-badge"/></a>
-<a href="https://github.com/topics/css"><img alt="CSS" src="https://img.shields.io/badge/CSS%20-%23E34F26.svg?&style=for-the-badge"/></a>
-<a href="https://github.com/topics/javascript"><img alt="javascript" src="https://img.shields.io/badge/Javascript%20-%23E34F26.svg?&style=for-the-badge&logo=javascript&logoColor=white"/></a>
-<br>
-<br>
-<a href="https://github.com/Ayushparikh-code/Web-dev-mini-projects/edit/main/README.md"><img src="https://badges.frapsoft.com/os/v1/open-source.svg?v=103"></a>
-<a href="https://github.com/Ayushparikh-code/Web-dev-mini-projects/edit/main/README.md"><img src="https://img.shields.io/badge/Built%20by-developers%20%3C%2F%3E-0059b3"></a>
-<a href="https://github.com/Ayushparikh-code/Web-dev-mini-projects/edit/main/README.md"><img src="https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=yellow"></a>
-<a href="https://github.com/Ayushparikh-code"><img src="https://img.shields.io/badge/Maintained%3F-yes-brightgreen.svg?v=103"></a>
-<br>
-</div>
-<div align="center">
-  
-# ADD ANY WEB DEVELOPMENT MINI PROJECT !
-
-</div>
-<br>
-
-## HOW TO CONTRIBUTE ? 👷 
-
-**1.** Fork [this](https://github.com/Ayushparikh-code/Web-dev-mini-projects) repository.
-
-**2.** Clone the forked repository.
-
-```terminal
-git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects 
-```
-
-**3.** Navigate to the project directory.
-
-```terminal
-cd Web-dev-mini-projects
-```
-
-**4.**  MAKE A NEW FOLDER WITH YOUR PROJECT NAME INSIDE web-dev-mini-projects & Add your project files (eg: index.html ,style.css, script.js) inside that folder
-<br>
-
-**5.**  Also Add a README file in your project folder which consists of Description/screenshots about your project !
-          
```

Comment:
You've removed the readme of the repository. You have to create a readme for your own project. @Sulagna-Dutta-Roy

Reply:
@Sulagna-Dutta-Roy, Don't remove this, please,